torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1381,6 +1378,7 @@ def __init__(
         # These are mutually exclusive options
         assert not (result_dtype and promotes_int_to_float)
         assert not (result_dtype and promotes_int_to_int64)
+        assert not (result_dtype and complex_to_real)
```

Comment:
but `vector_norm` accepts `dtype` arg?

Reply:
`result_dtype` is used to specify when a function always returns a given dtype (e.g. `eq`).